### PR TITLE
English translation, first pass

### PR DESCRIPTION
**Pull request by mlc**
This patch:
- sets up the [jekyll-multiple-languages plugin](http://jekyll-langs.liaohuqiu.net/) (you'll have to re-run `bundle install` for this to work)
- offers a draft translation of the teaser page to English.

It does not:
- make links between languages yet (you'll have to know to go to http://support.refugeesemancipation.com/en/)
- offer much in the way of a guarantee that my translation is correct (I speak very little German and relied a lot on Google Translate, then cleaned it up a bit.)

Offering this as much for discussion as for anything else; if there's interest, I'll clean it up further and we can go from there.

Thanks!

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html class="no-js" lang="de">
+<html class="no-js" lang="{{ page.language }}">
```
Comment (srenatus) [inline _includes/head.html:2]: This is not yet set by `index.md` or `index.en.md`, is it? (Might be some magic that I didn't notice. At least it's not in the index.html built by Jekyll.)

Comment (srenatus) [inline _includes/head.html:2]: Nevermind, forgot to update my config.yml. (Sorry for the noise.)

**Comment by plutonik-a** [comment]
> It does not make links between languages yet (you'll have to know to go to http://support.refugeesemancipation.com/en/)

We can provide a switch button which links to the en | de version, which is basically what we want, right?

**Comment by line-o** [comment]
That switch button needs to have the appropriate logic behind it. Is it enough to just remove the two character folder name or adding it?

**Comment by srenatus** [comment]
At least for now, there's only one page, so something like

```
{% if page.is_default_language %}
<!-- switch to english -->
{% else %}
<!-- switch to german -->
{% endif %}
```

in `_layouts/frontpage.html` would do (in [this case](https://github.com/refugee-emancipation/crowd-funding/commit/ba4953c2424908fcb4e80f88cdde88454896acd7)) or a logic-less switch in `frontpage.html` (to english) and in `frontpage.en.html` (to german).

But in general, judging from [the docs](http://jekyll-langs.liaohuqiu.net/), dropping/adding the prefix should work:

> ## Build result
> 
> The default language will build into destination directory as usually, _site, for example.
> The other language (mark as: $lang), will build into the subdirectory /$lang in the destination directory.
> For example: _site/$lang. If $lang is en, will be built into _site/en.

LGTM first? :)

**Comment by srenatus** [comment]
@mlc: Thank you very much for your contribution :smiley: -- it's been incorporated into the [multilang](https://github.com/refugee-emancipation/crowd-funding/tree/multilang) branch, to be merged soon. (I suppose this PR could thus be closed?)

**Comment by plutonik-a** [comment]
@mlc Of course, from me too a big thank you for your awesome PR.
@srenatus This PR will be closed automatically when branch 'multilang' is merged into master.

**Comment by mlc** [comment]
sounds good, thanks everyone!

**Pull request merged**